include/envoy/stats/stats.h

```diff
@@ -136,6 +136,7 @@ class Gauge : public Metric {
   virtual void set(uint64_t value) PURE;
   virtual void sub(uint64_t amount) PURE;
   virtual uint64_t value() const PURE;
+  virtual void setParentValue(uint64_t) {}
```

Comment:
nit: I think it would be better if this were pure? Might be also worth a comment since not obvious like the others?

Reply:
yup, was just doing the minimal edit to get a sense of direction. fixed.